lib/WorseReferenceFinder/Tests/Unit/WorseReflectionDefinitionLocatorTest.php

```diff
@@ -322,6 +322,20 @@ public function testLocatesCase(): void
         $this->assertTypeLocation($location->first(), 'FoobarEnum.php', 24, 33);
     }
 
+    public function testLocatesEnumConst(): void
+    {
+        if (!defined('T_ENUM')) {
```

Comment:
Can drop this as the minimum version is 8.1 (which includes enums)

Reply:
I dropped all checks for `defined('T_ENUM')`, is it ok?